packages/ui/Marquee/useMarqueeController.js

```diff
@@ -193,7 +196,7 @@ const useMarqueeController = (props) => {
 	 */
 	const handleComplete = useCallback((component) => {
 		const complete = markReady(component);
-		if (complete) {
+		if (complete && !component.contentFits) {
```

Comment:
I think we can have this change in develop branch, but we need to discuss when we make a change to deployed one.

Reply:
Okay I agree.